docs/docs/governance/cross-chain-governance.mdx

```diff
@@ -0,0 +1,156 @@
+4. Go to Connext module > write > `setTarget` with the Delay address
+5. Get the Connext module address then remove the Connext module
+6. Go to Delay > write > `enableModule` with the Connext module address
+
+## Using scripts 
+
+
+- Deploy the Delay module contract (see [this guide](https://github.com/gnosis/zodiac-modifier-delay/blob/main/docs/setup_guide.md)) 
+(NB: you can use the `deploy_multisig_mods` script in the `smart-contracts` folder to help you parse the commands)
+- add the `delayMod` to the `bridge.modules` object in the `networks` package
+- Deploy the Connext module contract (see [this guide](https://github.com/gnosis/zodiac-modifier-delay/blob/main/docs/setup_guide.md)) (NB: you can use the `deploy_multisig_mods` script to help you parse the commands)
+- add the `connextMod` to the `bridge.modules` object in the `networks` package
+- send the `proposals/enableModule.js` to the multisig on the chain to check settings and activate the module there
+- NB: make sure that `delayMod.target` is set to `safe.address` and `connextMod.target` is set to `delayMod.address`
+
+Now the multisig can receive calls from the bridge through the `connextMod` that will pass the call to the `delayMod` which will put it in cooldown.
+
+## Write a multichain DAO proposal
+
+We have a parser for DAO proposals that relies on object formatted as follow :
+
+```jsx
+return {
+  proposalName, // title of the proposal
+  calls: [
+    {
+      contractAddress: bridgeAddress, // the address of the Connext bridge
+      contractNameOrAbi: abiIConnext, // Connext Abi of `xcall`
+      functionName: 'xcall', // standard Connext function for bridge call
+      functionArgs: [
+        destDomainId, // the Domain ID of the receiving chain
+        destMultisigAddress, // the safe address on the receiving chain
+        ADDRESS_ZERO, // asset
+        ADDRESS_ZERO, // delegate
+        0, // amount
+        30, // slippage
+        moduleData, // the calldata to be executed on the receiving chain
+      ],
+    },
+  ],
+}
+```
+
+Once the proposal is correctly parsed, we can send it to the DAO
+
+```jsx
+yarn hardhat gov:submit --gov-address <governor> --proposal <proposal-filepath> --network gnosis <arguments>
+```
+
+Here the `<arguments>` passed to the cli as POSIT args are passed to the proposal script. See an example in `proposals/006-cross-bridge-proposal.js`
+
+```jsx
+yarn hardhat gov:submit --gov-address 0xE85696a3419162452e6925816D8073374e4190b7 --proposal proposals/006-cross-bridge-proposal.js --network gnosis 137 0xfa2709Aa98F051c4190d70dE38F7c7A330c60ab7 0x2411336105D4451713d23B5156038A48569EcE3a
+```
+
+## Executing a call
+
+Once a call has pass the cooldown delay, it can be executed by anyone (the addressed submitting the transaction does not need to be a signer on the multisig) via the `executeNextTx` in the Delay mod.
```

Comment:
```suggestion
Once a call has passed the cooldown delay, it can be executed by anyone (the addressed submitting the transaction does not need to be a signer on the multisig) via the `executeNextTx` in the Delay mod.
```